### PR TITLE
Add security reporting document

### PR DESCRIPTION
**Pull request by elahrvivaz**

### CODE REVIEW

**Comment by elahrvivaz** [comment]
rendered doc can be viewed here: https://github.com/elahrvivaz/geomesa/blob/fcr_security_md/SECURITY.md

**Comment by elahrvivaz** [comment]
based on the eclipse RDF4j project, which was indicated as a good template to follow: https://github.com/eclipse/rdf4j/blob/main/SECURITY.md

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+GeoMesa follows the [Eclipse Vulnerability Reporting Policy](https://www.eclipse.org/security/policy.php).
+Vulnerabilities are tracked by the Eclipse security team, in cooperation with the GeoMesa committers. Fixing
+vulnerabilities is done by the GeoMesa committers, with assistance and guidance from the security team.
+
+## Reporting a Vulnerability
+
+In the case of a suspected vulnerability, do not use the GeoMesa public issue tracker. Instead, contact the
```
Comment (jnh5y) [inline SECURITY.md:9]: Should we also say that folks should not use the mailing list or Gitter?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+[Eclipse Security Team](mailto:security@eclipse.org) directly.
```
Comment (elahrvivaz) [inline SECURITY.md:10]:
```suggestion
In the case of a suspected vulnerability, do **not** use any public GeoMesa forum (mailing list, chat or issue tracker).
Instead, contact the [Eclipse Security Team](mailto:security@eclipse.org) directly.
```

**Pull request merged**